RecetarioExpo/src/data/index.js

```diff
@@ -0,0 +1,62 @@
+const images = [
+    require('../assets/images/avocado_toast.jpg'),
+    require('../assets/images/banderilla.jpg'),
+    require('../assets/images/hotcakes.jpg'),
+    require('../assets/images/nodles.jpg'),
+    require('../assets/images/fish.jpg'),
+    require('../assets/images/ham_snacks.jpg'),
+    require('../assets/images/pasta.jpg'),
+    require('../assets/images/pizza.jpg'),
+    require('../assets/images/spaguetti.jpg'),
+    require('../assets/images/vegetarian_pie.jpg'),
+    require('../assets/images/vegetarian_toast.jpg'),
+];
+
+const data = [
+    {
+        title: 'Avocado Toast',
+        src: images[0],
+    },
+    {
+        title: 'Banderilla',
+        src: images[1],
+    },
+    {
+        title: 'Hot Cakes',
+        src: images[2],
+    },
+    {
+        title: 'Nodles',
+        src: images[3],
+    },
+    {
+        title: 'Fish',
+        src: images[4],
+    },
+    {
+        title: 'Ham Snacks',
+        src: images[5],
+    },
+    {
+        title: 'Pasta',
+        src: images[6],
+    },
+    {
+        title: 'Pizza',
+        src: images[7],
+    },
+    {
+        title: 'Spaguetti',
+        src: images[8],
+    },
+    {
+        title: 'Vegetarian Pie',
+        src: images[9],
+    },
+    {
+        title: 'Vegetarian Toast',
+        src: images[10],
+    },
+];
+
+export default data;
```

Comment:
As a rule of the Style Guide, always leave a blank line at the end of each file.